cmd/build-uki.go

```diff
@@ -93,7 +93,8 @@ func NewBuildUKICmd() *cobra.Command {
 				return fmt.Errorf("keys directory does not exist: %s", keysDir)
 			}
 			// Check if the keys directory contains the required files
-			requiredFiles := []string{"db.der", "db.key", "db.auth", "KEK.der", "KEK.auth", "PK.der", "PK.auth", "tpm2-pcr-private.pem"}
+			//requiredFiles := []string{"db.der", "db.key", "db.auth", "KEK.der", "KEK.auth", "PK.der", "PK.auth", "tpm2-pcr-private.pem"}
```

Comment:
hwo itts the pcr-private key not needed anymore? You cannot sign the measurements without it?

Reply:
you are right, this one is needed indeed. I'll keep the rest of them in the check for now. This is kind of irrelevant to this PR.

We only have to make sure the "der" files and the "auth" files, include the same certificats. Currently only the "auth" files include the custom certs (see my other comment below).